### PR TITLE
GitHub Pipeline updated for the functional test cases.

### PR DESCRIPTION
**Pull request by HariKarthykAAPNA**

### CODE REVIEW

File: src/store/Modules/Theme.ts [hunk 1]
```diff
@@ -123,9 +124,10 @@ export default class Connections extends VuexModule {
       this.context.commit("updateGlobalTheme", { data, themeIdx })
     } catch (err) {
       console.log(err)
+      this.context.commit("updateGlobalTheme", {})
```
Comment (ankitvats1) [inline src/store/Modules/Theme.ts:127]: Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.

File: src/store/Modules/Theme.ts [hunk 2]
```diff
@@ -111,9 +111,10 @@ export default class Connections extends VuexModule {
       this.context.commit("addGlobalThemes", data)
     } catch (err) {
       console.log(err)
+      this.context.commit("addGlobalThemes", {})
```
Comment (ankitvats1) [inline src/store/Modules/Theme.ts:114]: Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.

File: src/store/Modules/Queries.ts [hunk 3]
```diff
@@ -104,9 +105,10 @@ export default class Queries extends VuexModule {
       this.context.commit("updateGlobalQuery", data)
     } catch (err) {
       console.log(err)
+      this.context.commit("updateGlobalQuery", {})
```
Comment (ankitvats1) [inline src/store/Modules/Queries.ts:108]: Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.

File: src/store/Modules/Queries.ts [hunk 4]
```diff
@@ -77,10 +77,11 @@ export default class Queries extends VuexModule {
       this.context.commit("addGlobalQueries", data)
     } catch (err) {
       console.log(err)
+      this.context.commit("addGlobalQueries", {})
```
Comment (ankitvats1) [inline src/store/Modules/Queries.ts:80]: Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.

File: src/store/Modules/Projects.ts [hunk 5]
```diff
@@ -617,11 +622,12 @@ export default class Projects extends VuexModule {
       this.context.commit("updateQuery", data)
     } catch (err) {
       console.log(err)
+      this.context.commit("updateQuery", {})
```
Comment (ankitvats1) [inline src/store/Modules/Projects.ts:625]: Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.

File: src/store/Modules/Projects.ts [hunk 6]
```diff
@@ -587,10 +591,11 @@ export default class Projects extends VuexModule {
       this.context.commit("updateConnection", connectionData)
     } catch (err) {
       console.log(err)
+      this.context.commit("updateConnection", {})
```
Comment (ankitvats1) [inline src/store/Modules/Projects.ts:594]: Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.

File: src/store/Modules/Projects.ts [hunk 7]
```diff
@@ -572,10 +575,11 @@ export default class Projects extends VuexModule {
       this.context.commit("updateTheme", themeData)
     } catch (err) {
       console.log(err)
+      this.context.commit("updateTheme", {})
```
Comment (ankitvats1) [inline src/store/Modules/Projects.ts:578]: Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.

File: src/store/Modules/Projects.ts [hunk 8]
```diff
@@ -529,15 +529,18 @@ export default class Projects extends VuexModule {
...
-      this.context.commit("addInitialProjects", data)
-    } else {
-      this.context.commit("openSampleDataPopup")
+    try {
+      if (projects.length > 0) {
+        this.context.commit("addInitialProjects", data)
+      }
+    } catch (err) {
+      console.log(err)
+      this.context.commit("addInitialProjects", {})
```
Comment (ankitvats1) [inline src/store/Modules/Projects.ts:538]: Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.

File: src/store/Modules/Connections.ts [hunk 9]
```diff
@@ -98,9 +99,10 @@ export default class Connections extends VuexModule {
       this.context.commit("updateGlobalConnection", { data, connectionIdx })
     } catch (err) {
       console.log(err)
+      this.context.commit("updateGlobalConnection", {})
```
Comment (ankitvats1) [inline src/store/Modules/Connections.ts:102]: Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.

File: src/store/Modules/Connections.ts [hunk 10]
```diff
@@ -85,10 +85,11 @@ export default class Connections extends VuexModule {
       this.context.commit("addGlobalConnections", data)
     } catch (err) {
       console.log(err)
+      this.context.commit("addGlobalConnections", {})
```
Comment (ankitvats1) [inline src/store/Modules/Connections.ts:88]: Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.

**Review by ankitvats1 — CHANGES_REQUESTED** [review]

**Review by ankitvats1 — DISMISSED** [review]

**Review by Rahul139-stack — DISMISSED** [review]

**Pull request merged**